sdk/vision/azure-ai-vision-imageanalysis/src/samples/java/com/azure/ai/vision/imageanalysis/pom.xml

```diff
@@ -0,0 +1,28 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>azure.ai.vision.imageanalysis.samples</groupId>
+  <artifactId>image-analysis-quickstart</artifactId>
+  <version>0.0</version>
+  <dependencies>
+    <!-- https://mvnrepository.com/artifact/com.azure/azure-ai-vision-imageanalysis -->
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-ai-vision-imageanalysis</artifactId>
+      <version>1.0.0-beta.1</version>
+    </dependency>
+    <!-- https://mvnrepository.com/artifact/org.slf4j/slf4j-api -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+      <version>2.0.7</version>
+    </dependency>
+    <!-- https://mvnrepository.com/artifact/org.slf4j/slf4j-simple -->
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-simple</artifactId>
+      <version>2.0.7</version>
+    </dependency>
```

Comment:
This POM could be removed and these dependencies added as test dependencies in the SDK POM. Samples are compiled during test compilation.

Reply:
@alzimmermsft I would like to offer developers the option to build the samples and test them against the public package. Therefore I have this separate pom.xml next to the samples source code. The samples README.md shows how to build and run all or a single sample.
Of course I would also like the ADO pipeline to automatically build the samples after building the SDK and running tests. I'll look at the pipline to make sure samples are built. The room pom.xml already has >slf4j-simpl (which is required during compile time). slf4j-api package is required during run-time if you turn on logging. Some verification tool does not allow me to add this dependency in the root pom.xml. I'm assuming for a good reason. 
So bottom line: I prefer keeping things as is, while validating that indeed samples are compiled as part of ADO pipeline. I'll keep this comment open until I verify.